src/lib/Hydra/Controller/User.pm

```diff
@@ -226,6 +272,29 @@ sub github_redirect :Path('/github-redirect') Args(0) {
     $c->res->redirect("https://github.com/login/oauth/authorize?client_id=$client_id&scope=user:email");
 }
 
+sub oidc_redirect :Path('/oidc-redirect') Args(0) {
+
+    my ($self, $c) = @_;
+
+    my $after = "/" . $c->req->params->{after};
+
+    $c->res->cookies->{'after_oidc'} = {
```

Comment:
I saw this in GitHub as well and I really don't know why we would do this. We already have a session so why not do:
```
$c->session->{oidc_after} = $after;
```
You already do this for the state (which is what GitHub is not doing right now afaik :eyes:)

Reply:
Yes, this is just taken directly from the other auth methods. We did discuss that it should also probably be a single session variable regardless of the auth method. I'd prefer to fix this later though.